rust/src/tx_builder.rs

```diff
@@ -96,7 +96,8 @@ fn fake_full_tx(tx_builder: &TransactionBuilder, body: TransactionBody) -> Resul
         0 => None,
         _x => {
             // TODO: figure out how to populate fake witnesses for these
-            return Err(JsError::from_str("Script inputs not supported yet"))
+            // return Err(JsError::from_str("Script inputs not supported yet"))
```

Comment:
Was this meant to just be commented out and not removed?

Reply:
Yes for now, will get rid of it for the proper release version